drafts/te-types-update/draft-ietf-teas-rfc8776-update.md

```diff
@@ -305,12 +305,18 @@ switching-capabilities:
 > A base YANG identity for supported interface switching capabilities as defined in {{?RFC3471}}.
 
 resource-affinities-type:
-
+exx
```

Comment:
what is exx?

Reply:
Fixed in https://github.com/tsaad-dev/te/pull/252/commits/c0c66369eedb66b4a66bb27627f37a8aa5576953